README.md

```diff
@@ -29,4 +35,4 @@ A simple example: You are running Plex on an Ubuntu server and set Plex up to li
 
 **Please note:** You need to either quote the path to the config or escape the contained spaces.
 
-In order to automate the process, create a cronjob or other type of scheduled task in order to run the script regularly. Just keep in mind that your Plex server will be unavailable for a few seconds when the script restarts the Plex service after an IPv6 address change, so choose times to run the script accordingly.
\ No newline at end of file
+In order to automate the process, create a cronjob or other type of scheduled task in order to run the script regularly. Just keep in mind that your Plex server will be unavailable for a few seconds when the script restarts the Plex service after an IPv6 address change, so choose times to run the script accordingly.
```

Comment:
Well, if your IPv6 changes dynamically and it is an requirement that the current IPv6 is reflected in the plex custom url setting than there is no need to "choose times to run the script accordingly". The moment when the IPv6 changes the custom url becomes invalid and external connections should not be possible from that moment. So you should run the script regularly across the whole day.

Reply:
Yes, you are correct. I currently run the script every two minutes.

But I do not see how this section of the README is relevant here.